app/views/rent_mailer/rent_email.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Rent created</h1>
+
+<p>Fecha de creacion: <%= Date.current.strftime('%d-%m-%Y') %></p>
+
+<h3>Detalles del libro</h3>
+<ul>
+  <li>Title: <%= @rent.book.title %></li>
+  <li>Genre: <%= @rent.book.gender %></li>
+  <li>Author: <%= @rent.book.author %></li>
+  <li>Year: <%= @rent.book.year %></li>
+  <li>Editor: <%= @rent.book.editor %></li>
+</ul>
+
+<p>Rent start: <%= @rent.rent_start.strftime('%d-%m-%Y') %></p>
+<p>Rent end: <%= @rent.rent_end.strftime('%d-%m-%Y') %></p>
```

Comment:
Oh !"#%...HTML, @blacksam07 I live this one to you xD

Reply:
is the beautiful erb